kolibri/plugins/coach/assets/src/views/reports/table-cells/header-cell.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <th scope="col" :style="{ textAlign: align }">
+  <th @click="setSortOrder" scope="col" :style="{ textAlign: align }">
     <button v-if="sortable" class="header-text" @click="$emit('click')">
```

Comment:
We should use the button's click event rather than attaching to the table header.

For context, in the old implementation the table header had state set completely from the outside and just emitted click events. I think tying this to vuex is fine for now, and we can remove the click event.